src/test/java/com/moodybluez/enterprise/EnterpriseApplicationTests.java

```diff
@@ -1,18 +1,17 @@
 package com.moodybluez.enterprise;
```

Comment:
Updated and Optimized the tests a little bit. Was also necessary since I made changes in DTOs and created separate DAO layer. Good thing is they still pass.